app/stores/WalletsStore.js

```diff
@@ -180,7 +180,7 @@ export default class WalletsStore extends Store {
   isValidPrivateKey = () => { return true; }; // eslint-disable-line
 
   @action refreshWalletsData = async () => {
-    if (this.stores.networkStatus.isConnected) {
+    if (this.stores.networkStatus.isConnected && this.stores.networkStatus.isSynced) {
```

Comment:
I guess we only need `isSynced` here. Or maybe we should check the sync percentage to be more than 99%? With 15-second slots syncing is very often less than 100% so the syncing spinner is spinning.